demos/java/spring/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.resources.static-locations=file:<YOUR-PATH>/developer-authentication-demos/theme
```

Comment:
I replaced the URL below and it works. So please change the same here and it is ready to merge.
 
```
file:///<YOUR-PATH>/developer-authentication-demos/theme
```
Thanks @romain325